src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```
@@ -4745,6 +4745,7 @@ components:
                 - kind: "U32"
                   value: "1"
     StateKeyValueStoreDataRequestKeyItem:
+      description: It is required to provide either key_hex or key_json property.
```

Please wrp property names in backticks to improve formatting.

---

It sounds a little clunky. Just the following is fine I think:
```suggestion
      description: Provide either `key_json` or `key_hex`.
```
.